core/storage/job/gae_models.py

```diff
@@ -143,3 +143,55 @@ class ContinuousComputationModel(base_models.BaseModel):
     # The time at which a halt signal was last sent to this batch job, in
     # milliseconds since the epoch.
     last_stopped_msec = ndb.FloatProperty(indexed=True)
+
+
+# Allowed transitions: processing --> completed --> archived
+# or processing --> failed.
+QUERY_STATUS_PROCESSING = "processing"
```

Comment:
USER_QUERY_STATUS_*, to distinguish it from other types of queries.

feconf might be a better place for these, especially since you're referring to them from other parts of the codebase.